api/index.js

```diff
@@ -12,56 +12,46 @@ var errors = require('./lib/errors');
 var serverLib = require('./lib/server-lib');
 var remote = require('./lib/remote');
 
-module.exports = {
-  info: {
-    serverStatus: Info.serverStatus,
-    isConnected: Info.isConnected,
-    fee: Info.fee,
-    uuid: Info.uuid
-  },
+function RippleAPI() {
+  this.remote = remote;
+}
```

Comment:
All instances of RippleAPI would be using a remote singleton. We need to allow each instance of RippleAPI to have it's own instance of remote.

    var remote = require('remote');    // this is a remote singleton
    var Info = require('./info')(remote);    // Info is now tied to a single instance of remote